src/main.jsx

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import ReactDOM from 'react-dom/client'
 import App from './App.jsx'
 import ErrorPage from "./components/ErrorPage.jsx"
+import Footer from "./components/Footer.jsx"
 import { createBrowserRouter, RouterProvider } from 'react-router-dom'
 
 const router = createBrowserRouter([
   {
     path: "/",
     element: <App/>,
-    errorElement: <ErrorPage/>
+    errorElement: <ErrorPage/>,
+    footerElement: <Footer/>
```

Comment:
Hey @QH89, I think we Footer will actually be imported to `App.jsx` rather than `Main.jsx`